schemas/events/v1/events.json

```diff
@@ -67,6 +67,9 @@
         {
           "$ref": "../../core/v1/error.json"
         },
+        {
+          "$ref": "../../core/v1/notification.json"
+        },
```

Comment:
This part is kind of tricky - unsure if i understood correctly, but a `notification` would expect to augment the event send by the app it self.

This lets `notifications` know if they need a special processing options for the email of this particular event.
In the current situation there won't be any `advisor` event - only the notification. 

Should the `notification` schema be itself part of the other app schemas that are supported by notifications? 
i.e. `advisor-recommendation` extend (via a property `notification_settings` or any other mechanism)?

Reply:
Generally, an application should be able to just use the notification schema directly. This would be appropriate when the application and event type together are sufficient. For example "cost management's monthly report is ready".

If the notification needs to include more information than is described by just event type/application, then that app needs to define its own dataschema, and needs to include a `notification_recipients` field. An example of this might be "Vulnerability detected the X critical CVE on Y number of systems".

Reply:
This makes sense to me, we haven't seen such an scenario in our use cases - but seems valid to me.

If they want to further configure how to manage their notification, they can optionally extend the new schema (to have the `notification_recipient` as part of their schema)